docs/_includes/intrinsic_attributes.adoc

```diff
@@ -0,0 +1,517 @@
+|Any (value not used)
+|<<footer-docinfo-files>>
+
+
+|noxmlns
+|Remove the namespace from the DocBook 5 XML output.
+|(not set - namespace will be added)
+|Any (value not used)
+|<<docbook>>
+
+
+|prewrap
+|Wrap wide code listings. (Sets the default behavior only; you can still switch off wrapping on specific listings).
+|Empty - code listing will wrap, not scroll.
+|Any (value not used)
+|<<to-wrap-or-to-scroll>>
+
+
+|scriptsdir
+|Sets the path to JavasScript files
+|./javascripts
+|Path
+|<<setting-attributes-on-a-document>>
+
+
+|sectanchors
+|If set, adds an anchor (empty link) before the section title.
+|Not set - no anchors.
+|Any (value not used)
+|<<anchors>>
+
+
+|sectids
+|If set, synthesises IDs for any sections that do not already have one.
+|Empty string.
+|Any (value not used)
+|<<auto-generated-ids>>
+
+
+|sectlinks
+|Turns section titles into links.
+|Not set.
+|Any (value not used)
+|<<links>>
+
+
+|sectnumlevels
+|controls the depth of section numbering
+|3
+|0,1,2,3,4,5
+|<<numbering-depth>>
+
+
+|showtitle
+|Toggles the display of an embedded document’s title.
+|Not set.
+|Any (value not used)
+|<<document-title>>
+
+
+|source-highlighter
+|Source code highlighter to use.
+|Not set.
+|coderay, highlightjs, prettify, pygments
+|<<source-code-blocks>>
+
+
+|source-language
+|Set the default language for source code listings.
+|Not set.
+|Defined by the source highlighter in use, but includes at least java, c, ruby, xml.
+|<<source-code-blocks>>
+
+
+|stem
+|Set the processor used to render mathematics.
+|asciimath
+|asciimath, latexmath.
+|<<stem-in>>
+
+
+|toc
+|Switches the table of contents on, and defines its location.	
+|auto
+|auto, left, right, macro, preamble
+|<<user-toc>>	
+
+
+|toclevels
+|Maximum section level to display.
+|2
+|1,2,3,4,5
+|<<user-toc>>	
+
+
+|webfonts
+|Control how webfonts are referenced by the default stylesheet.
+|Set.
+|
+|<<applying-a-theme>> and link:https://github.com/asciidoctor/asciidoctor.org/issues/410[issue #410]
+
+
+5+^s|Information
+
+// Need to test this.
+// My guess is that it is  if ad finds a compatibility file. 
+|compat-mode
+|If set, makes the processor run in compatibility mode. 
+|Not set - runs in the 'modern' mode.
+|Any (value not used)
+|
+
+|embeddable
+|Set when the `header_footer` option has been disabled (i.e. the document is being converted to an embeddable document).
+|NA
+|NA - read only
+|
+
+|====
+
+[1] The default value isn't necesarily the value you will get by entering `\{name}`; it may be the "fallback value" which Asciidoctor uses if `name` is not defined. 
+The effect is the same either way.
+
+The opposite of header attributes, _in-document attributes_ are designed to be set in the body of the document. 
+(You could set them in the header, but then they will apply for the whole document which is not ussually what you want.)
+They are visible from the point they are set, until they are unset or assigned another value.
+
+.In-document attributes
```

Comment:
Perhaps we should say "Universal attributes" instead of "in-document". In mailinglist posts, I've often referred to them as "document flow attributes" (or something to that effect), meaning "in the flow of the document".